### ISSUE TITLE
Unbalanced brackets in file mok.c is causing compile error 

### ISSUE THREAD

**Issue by JimGaluska** [rhboot/shim#168]
Function maybe_mirror_one_mok_variable in file mok.c is missing a close bracket.  This is causing a compile error:

```c
275   static EFI_STATUS nonnull(1)
276   maybe_mirror_one_mok_variable(struct mok_state_variable *v, EFI_STATUS ret)
277   {
278   	EFI_STATUS efi_status;
279   	if (v->rtname) {
280    		if (v->flags & MOK_MIRROR_DELETE_FIRST)
281    			LibDeleteVariable(v->rtname, v->guid);
282
283    		efi_status = mirror_one_mok_variable(v);
284    		if (EFI_ERROR(efi_status)) {
285    			if (ret != EFI_SECURITY_VIOLATION)
286    				ret = efi_status;
287    			perror(L"Could not create %s: %r\n", v->rtname,
288    			       efi_status);
289    	}
290    	return ret;
291    }
```

**Comment by diabonas** [comment]
I noticed that as well, see my pull request #166 for a fix.

**Comment by JimGaluska** [comment]
Thanks for the quick response.  I looked through the Issues, but I didn't think to look through the Pull Requests to see if this has already been addressed.

Would you like me to close this issue now, or should I wait until Pull Request #166 is merged

**Comment by diabonas** [comment]
I suggest leaving the issue open to increase visibility for other users. I have edited my PR description so that this issue will be closed automatically once #166 is merged.

**Comment by JimGaluska** [comment]
I confirmed this this issues is fixed.  I was able to pull commit ID cd20ce4, compile, and boot a computer securely with this shim.

Thank you!
